docker/utils/update_config_directory.sh

```diff
@@ -29,7 +29,7 @@ copy_package_file() {
 # source robot environment
 source /run/husarion/robot_config.env
 
-copy_package_file husarion_ugv_bringup config/joy2twist_${ROBOT_MODEL_NAME}.yaml
+copy_package_file husarion_ugv_teleop config/joy2twist.yaml
```

Comment:
We should use the configurations with the names because the robots have different max velocities.
```suggestion
copy_package_file husarion_ugv_teleop config/joy2twist_${ROBOT_MODEL_NAME}.yaml
```